sphinx_airflow_theme/sphinx_airflow_theme/footer.html

```diff
@@ -0,0 +1,136 @@
+{#
```

Comment:
Can you please update the footer? There are some minor changes (new links, smaller font in the bottom part)